tests/acceptance/scripts/build.sh

```diff
@@ -14,7 +14,7 @@ fi
 
 count=0
 while [[ 3 -gt $count ]]; do
-    docker build . -f tests/terraform/scripts/Dockerfile.build -t rke2-tf-"${TRIM_JOB_NAME}""${BUILD_NUMBER}"   
+    docker build . -f tests/rke2tf/shared/scripts/Dockerfile.build -t rke2-tf-"${TRIM_JOB_NAME}""${BUILD_NUMBER}"
```

Comment:
Should be `tests/acceptance/scripts` as there are no dir called `rke2tf/shared`

Reply:
done tks